integration/test/agent_util_linux.go

```diff
@@ -32,6 +32,19 @@ func CopyFile(pathIn string, pathOut string) {
 	log.Printf("File : %s copied to : %s", pathIn, pathOut)
 }
 
+func DeleteFile(filePathAbsolute string) error {
```

Comment:
Perhaps we could leverage https://ieftimov.com/posts/testing-in-go-clean-tests-using-t-cleanup?

Reply:
Can't because I'm running only a one test *testing.T function in order to create a nice summary. If that happens we only clean up once for the entire suite which is not the behavior you wanted